b2sdk/bucket.py

```diff
@@ -292,6 +292,36 @@ def list_parts(self, file_id, start_part_number=None, batch_size=None):
         """
         return self.api.list_parts(file_id, start_part_number, batch_size)
 
+    def list_file_versions(self, file_name, fetch_count=None):
+        """
+        Lists all of the versions for a single file.
+
+        Depending on a Bucket's settings many versions may exist for the same filename.
```

Comment:
As far as I know, there is no bucket setting in B2 that would disable this capability.

Reply:
A bucket's lifecycle settings control how many versions of a file to keep, one of the options is to only keep the most recent version, though admittedly that does still mean more than one version can exist, as I understand it older versions get removed periodically.

Reply:
I'll change the wording a little to see if it makes more sense.

Reply:
I guess this just overcomplicates matters, I'll just remove it.

Reply:
👍